recipes/powerlaw/license.txt

```diff
@@ -0,0 +1,23 @@
+#The MIT License (MIT)
```

Comment:
Since the license is included in the source code, I do not think it is necessary to manually package a separate license file that was created here rather than by the upstream source. Please remove this file

Reply:
License file removed